cmd/shutdown_windows.go

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------
```

Comment:
Shouldn't this also have the +build windows header?

Reply:
Adding the suffix `_windows.go` to the filename achieves the same thing: https://golang.org/cmd/go/#hdr-Build_constraints